src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -444,7 +445,8 @@ private static void InitializeGetEnumerableCache()
 
         internal static bool IsTypeEnumerable(Type type)
         {
-            if (type == null) { return false; }
+            if (type == null)
+            { return false; }
```

Comment:
I mean such style issues. We should either fix or revert.

Reply:
I'll trim these out for now to keep things simple. 👍